torch/nativert/common/Conv.cpp

```diff
@@ -0,0 +1,51 @@
+#include <torch/nativert/common/Conv.h>
+
+#include <cerrno>
+#include <cstdlib>
+#include <string>
+
+namespace torch::nativert {
+
+template <>
+std::optional<int64_t> tryTo<int64_t>(std::string_view symbol) {
+  // TODO Using strtoll for portability. Consider using std::from_chars in the
+  // future.
+  std::string symbol_str(symbol);
```

Comment:
since we need to copy the input if we don't have a guaranteed null terminator, I would provide efficient overloads of this function taking `const std::string&` and `const char*`. It should be easy to extend your tests to cover the overloads as well if you use the array-based approach I outlined.